src/topi/einsum.cc

```diff
@@ -265,13 +265,13 @@ class EinsumBuilder {
         auto ellipsis_shape = ellipsis_shape_.value();
         for (int i = 0; i < static_cast<int>(ellipsis_shape.size()); ++i) {
           reduction_axes->push_back(
-              IterVar(Range(0, ellipsis_shape[i]), Var("k"), IterVarType::kCommReduce));
+              IterVar(Range(0, ellipsis_shape[i]), Var("k", DataType::Int(64)), IterVarType::kCommReduce));
```

Comment:
May I ask if this data type annotation impact any other component?

Reply:
Good question, i didnt find any other failing tests so i assume this is ok. It seems needed to work with Relax since relax has a hard requirement for int64 itervars.

Reply:
We are already converting all indices to int64 automatically https://github.com/octoml/relax/blob/relax/python/tvm/relax/block_builder.py#L427